.github/actions/build-refs/action/index.ts

```diff
@@ -66,7 +66,31 @@ function latestTagPrefixFor(repo: Repo, variant: Variant): string[] {
 }
 
 export function latestTag(tagRefs: GitHubApiTag[]): Tag | null {
-  return (tagRefs.at(-1)?.ref as Tag | null | undefined) ?? null
+  if (tagRefs.length === 0) return null
+  
+  // Sort tags by version number (semantic versioning)
+  const sortedTags = tagRefs
+    .map(tag => tag.ref)
+    .sort((a, b) => {
+      // Extract version numbers from refs like "refs/tags/v1.19.4"
```

Comment:
this also needs to handle tags prefaced with `internal@`, like `internal@1.2.0-alpha.0`. it also needs to handle semver tags. we probably should use https://www.npmjs.com/package/semver